src/popup/pages/Names/Claim.vue

```diff
@@ -89,10 +89,11 @@ export default defineComponent({
 
     const isNameValid = computed(() => name.value && checkAensName(`${name.value}${AENS_DOMAIN}`));
 
-    const nameFee = computed(() => convertToken(
-      TxBuilderHelper.getMinimumNameFee(`${name.value}${AENS_DOMAIN}`),
-      -AETERNITY_COIN_PRECISION,
-    ).toFixed(4));
+    const nameFee = computed(() => getMinimumNameFee(
+      `${name.value}${AENS_DOMAIN}` as AensName,
+    )
+      .shiftedBy(-AETERNITY_COIN_PRECISION)
+      .toFixed(4));
```

Comment:
`4` could be defined as constant

Reply:
It is used only within this file and there is a big chance is shouldn't be done like that. It may be a part of future aens update topic.